pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -165,104 +196,48 @@ func engineSetup() (Config, error) {
 	if err != nil {
 		return EngineConfig, err
 	}
+	logrus.Debug("Engine FQDN: ", EngineConfig.FQDN)
 
-	// Set c.URL with the API endpoint
-	EngineConfig.URL = fmt.Sprintf(
-		"https://%s/ovirt-engine/api",
-		EngineConfig.FQDN)
+	// By default, we set Insecure true
+	EngineConfig.Insecure = true
 
-	// Set PEM URL for Download
-	EngineConfig.PemURL = fmt.Sprintf(
-		"https://%s/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA",
-		EngineConfig.FQDN)
-	logrus.Debug("PEM URL: ", EngineConfig.PemURL)
+	// Set c.URL with the API endpoint
+	EngineConfig.URL = fmt.Sprintf("https://%s/ovirt-engine/api", EngineConfig.FQDN)
+	logrus.Debug("Engine URL: ", EngineConfig.URL)
 
-	// Simple http get to check if FQDN/URL are valid
-	err = checkURLResponse(EngineConfig.URL, true)
+	// Start creating HTTPResource struct for checking if Engine FQDN is responding
+	HTTPResource.skipVerify = true
+	HTTPResource.urlAddr = EngineConfig.URL
+	err = HTTPResource.checkURLResponse()
 	if err != nil {
 		return EngineConfig, err
 	}
 
-	// Check if CA is trusted locally
-	var importCACert bool = false
-
-	// Store if connection with Engine is secure
-	var ConnectionSecure bool = false
+	// Set Engine PEM URL for Download
+	EngineConfig.PemURL = fmt.Sprintf(
+		"https://%s/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA",
+		EngineConfig.FQDN)
+	logrus.Debug("PEM URL: ", EngineConfig.PemURL)
 
-	ConnectionSecure, err = checkCATrust(EngineConfig.FQDN)
+	// Create tmpFile to store the Engine PEM file
+	tmpFile, err := ioutil.TempFile(os.TempDir(), "engine-")
 	if err != nil {
-		return EngineConfig, err
-	}
-
-	if ConnectionSecure {
-		EngineConfig.Insecure = false
-	} else {
-		EngineConfig.Insecure = true
-		message := fmt.Sprintf("Would you like to import Engine CA cert locally from: %s ?", EngineConfig.PemURL)
```

Comment:
I don't understand why do we need to remove this logic and how is importCertIntoSystemPool is an improvement,
Do we have a place where we discussed it?
Can you maybe edit the commit message to explain?

Reply:
Only slack. I got a message saying, we can't use sudo in the installer. Based on that, we can't offer to users to import the CA from engine. Only import/load into the https request.